src/server.py

```diff
@@ -315,10 +315,22 @@ def get_tracker_bots(tracker_id) -> Response:
     return jsonify([e.serialize() for e in bots])
 
 
-@app.route("/api/trackers/<int:tracker_id>")
-def get_tracker(tracker_id: int) -> Response:
-    with model.database_connection() as conn:
-        tracker = model.Tracker.get_by_id(conn.cursor(), tracker_id)
+@app.route("/api/trackers/<id_or_dhash>")
+def get_tracker(id_or_dhash: str) -> Response:
+    tracker = None
+    if id_or_dhash.isdigit():
```

Comment:
I'd swap the order of the 2 if clauses. Sure - getting a SHA2 that consists of just digits is virtually impossible but it seems a bit more "correct"